src/main/java/it/pagopa/interop/probing/eservice/operations/repository/query/builder/ProducerQueryBuilder.java

```diff
@@ -18,22 +17,21 @@ public class ProducerQueryBuilder {
   @PersistenceContext
   private EntityManager entityManager;
 
-  public List<Producer> findAllProducersByProducerName(Integer limit, Integer offset,
+  public SearchProducerNameResponse findAllProducersByProducerName(Integer limit, Integer offset,
```

Comment:
I think the flow i snot correct. 
INPUT -> Request -> DTO -> Entity
OUTPUT Entity -> DTO -> Response

Rest Controller has to manage only Request and Response
Service has to manage only DTO
Repository has to manage only Entity

This a loose coupled exagonal architecture.

But I am an old fashion programmer nowadays.... almost as a boomer.

Reply:
Okay, you are right! I just pushed, let me know if the fix is ok